components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -122,6 +122,10 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
  ic_tod="0" sim_year="2000" glc_nec="0" use_crop=".false." >lnd/clm2/initdata_map/clmi.ICRUCLM45.ne4_oQU240.67a0f98.0001-01-04-00000.nc
 </finidat>
 
+<finidat hgrid="r0125"   maxpft="17" mask="oRRS15to5" use_cn=".false." ic_ymd="20000101" more_vertlayers=".false."
```

Comment:
I made a mistake here and the correct entry should be`hgrid="ne30np4"`.

Reply:
I don't see a mistake here, @bishtgauta. The file specified is interpolated to r0125 from ne30np; it is fine here.

Reply:
Based on the `*.ne30_oEC.*.nc`, I thought the land is on the same grid as the atmosphere (i.e. `ne30np4`)

Reply:
Yeah, one could easily get that impression. The filename becomes long; need to scroll to the right to see additional string attached to the filename.

Reply:
How about getting rid of `to_r0125` at the end and rename the file to be `20180716.DECKv1b_A3.ne30_r0125_oEC.edison.clm2.r.2010-01-01-00000.20200213.nc`?

Reply:
Good idea, @bishtgautam . Since the PR has been merged to next and the report is now back and well, I am going to merge this PR to master as is, then open a new one to make the change.